src/living/cssom/parsers/index.ts

```diff
@@ -950,3 +952,44 @@ export function shorthandSetter(
     }
   };
 }
+
+export function parserTransform(transformStr: string): DOMMatrixImpl {
```

Comment:
```suggestion
export function parseTransform(transformStr: string): DOMMatrixImpl {
```

And a parse function should not return a DOMMatrix